pconduit/v1/client/schema.go

```diff
@@ -32,23 +32,25 @@ type SchemaServiceClient struct {
 var _ pconduit.SchemaService = (*SchemaServiceClient)(nil)
 
 func NewSchemaServiceClient(cc *grpc.ClientConn) *SchemaServiceClient {
-	return &SchemaServiceClient{grpcClient: conduitv1.NewSchemaServiceClient(cc)}
+	return &SchemaServiceClient{
+		grpcClient: conduitv1.NewSchemaServiceClient(cc),
+	}
 }
 
 func (c *SchemaServiceClient) CreateSchema(ctx context.Context, request pconduit.CreateSchemaRequest) (pconduit.CreateSchemaResponse, error) {
-	ctx = internal.RepackConnectorTokenOutgoingContext(ctx)
+	createCtx := internal.RepackConnectorTokenOutgoingContext(ctx)
```

Comment:
Nitpick: I'd rather overwrite `ctx`, since we wrap the existing context and we don't want to use the old unwrapped context by mistake.

Reply:
I was thinking if that could have side effects on the caller (who'll get the new context too).

Reply:
The caller won't see anything, we're just overwriting our local variable here. The caller would only see it if they gave us a `*context.Context` and we overwrite it using `*ctx = ...`.

Reply:
Ah, right. :+1: